odom_callback_timeout: 60 # control loop will be cutoff when odometry latency is higher than this
vision_callback_timeout: 100 # control loop will be cutoff when vision latency is higher than this